composer.json

```diff
@@ -14,6 +14,8 @@
     ],
     "require": {
         "php": ">=7.1",
+
```

Comment:
No extra blank line please. 😂

Reply:
there was one before IIRC https://github.com/api-platform/core/blob/master/composer.json#L17

Reply:
I don't see why there should be...

Also, https://github.com/api-platform/core/blob/master/composer.json#L29 is in the wrong position for some reason. 😂😂